SU-SWS/stanford_paragraph_types
# Soe redesign

# READY FOR REVIEW

# Summary
- This adds two new paragraph types for SoE: the two column and a simplified text area with only an image insert.

# Needed By (Date)
ASAPP (As soon as practically possible)

# Urgency
Needs styles applied, particularly for the two column paragraph.

# Steps to Test

1. Enable Stanford Magazine
2. Create a new Stanford Magazine Article
3. Select and create at least one of each paragraph type
4. Verify it looks as expected

# Affected Projects or Products
- Stanford Magazine Article

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-1845
- Anyone who should be notified? @josephgknox, and @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
